content/content.tsx

```diff
@@ -35,7 +42,7 @@ async function initial() {
 
 async function fetchCSS() {
   // extension.js has some specific process if you fetch the css in the entry point of content script.
-  const cssUrl = new URL('~/styles/tailwind.css', import.meta.url)
+  const cssUrl = tailwindUrl
```

Comment:
👍 Cool, I didn't know it could work like this, and I think this way of writing it is much better, thanks.